src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs

```diff
@@ -593,11 +655,6 @@ private static unsafe int NativeCallback(QUIC_HANDLE* connection, void* context,
 
         try
         {
-            // Process the event.
```

Comment:
Honestly, why don't we add `ToString` to the event data structure and log it from one place?

Reply:
That change would need to go to the generated interop stubs from MsQuic then, right?

Reply:
Not necessarily, all the structs are partial, so we can add it to a different file. Or just define it as extension/static method.

Reply:
I don't think that would be beneficial. It may contains many field (we don't even use) and dumping everything is going to make logs hard to read IMHO. It will incur performance penalty and bring no value. 
If we feel strongly about this we can do it at at the Verbose level - we use it in Sockets for example to dump part of the actual data. 
I think using nameof() on interesting fields and consistent formatting may be the best

Reply:
It would be us coding and thus controlling what goes in.